scripts/windows-binary-identity.ps1

```
@@ -0,0 +1,112 @@
+    $start.StandardOutputEncoding = [Text.UTF8Encoding]::new($false)
+    $start.StandardErrorEncoding = [Text.UTF8Encoding]::new($false)
+    [void]$start.ArgumentList.Add('--version-json')
+    $process = [Diagnostics.Process]::Start($start)
+    if ($null -eq $process) {
+        throw "Could not start DefenseClaw identity input: $Path"
+    }
+    try {
+        $stdoutTask = $process.StandardOutput.ReadToEndAsync()
+        $stderrTask = $process.StandardError.ReadToEndAsync()
+        if (-not $process.WaitForExit($TimeoutSeconds * 1000)) {
+            try { $process.Kill($true) } catch {}
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)